pkg/sidecar/util_common.go

```diff
@@ -0,0 +1,52 @@
+package sidecar
+
+import (
+	"context"
+	"sync"
+
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// RunDaemon starts the main loop
+// Starts the workload API client to listen for new SVID updates
+// When a new SVID is received on the updateChan, the SVID certificates
+// are stored in disk and a restart signal is sent to the proxy's process
+func (s *Sidecar) RunDaemon(ctx context.Context) error {
```

Comment:
I think it would be more appropriate to place this method in `sidecar.go` in order to keep related functionalities grouped together.

Reply:
Yes, I agree